R/guide-colorbar.r

```diff
@@ -175,19 +192,18 @@ guide_train.colorbar <- function(guide, scale) {
     warning("colorbar guide needs continuous scales.")
     return(NULL)
   }
-
-
+  
   # create data frame for tick display
   breaks <- scale$get_breaks()
   if (length(breaks) == 0 || all(is.na(breaks)))
     return()
+    
```

Comment:
Can you please remove these minor changes that aren't related to your new functionality? That makes it easier for me to review.